src/app/features/admin/validators/connected-station.validator.ts

```diff
@@ -0,0 +1,11 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+import { StationsService } from '@core/services/stations/stations.service';
+
+export function connectedStationValidator(stationsService: StationsService): ValidatorFn {
```

Comment:
oh, i didn't know you could pass a service to the validator. thanks for the idea ❤️

Reply:
I'm not sure if this is the right thing